src/content/tools/pub/pubspec.md

```diff
@@ -498,7 +498,7 @@ works with any Dart SDK that's version 3.0.0 or higher:
 
 ```yaml
 environment:
-  sdk: ^3.0.0
+  sdk: ^3.2.0
```

Comment:
This no longer matches the above text on L497 now. Either update that or the snippet.

Reply:
```suggestion
  sdk: ^3.0.0
```